api/src/main/java/org/apache/gravitino/authorization/MetadataObjectChange.java

```diff
@@ -44,10 +45,11 @@ static MetadataObjectChange rename(
    * Remove a metadata entity MetadataObjectChange.
    *
    * @param metadataObject The metadata object.
+   * @param locationPaths The location paths of the metadata object.
```

Comment:
LocationPath is a weird name that will cause confusion.
In another PR, we already some some questions related to the difference between "location" and "path".
Better use location for one thing and path for another.

Reply:
I resubmit PR